project/project/project/metals.sbt

```diff
@@ -1,4 +1,4 @@
 // DO NOT EDIT! This file is auto-generated.
 // This file enables sbt-bloop to create bloop config files.
 
-addSbtPlugin("ch.epfl.scala" % "sbt-bloop" % "1.4.8")
+addSbtPlugin("ch.epfl.scala" % "sbt-bloop" % "1.4.8-19-4d9f966b")
```

Comment:
Out of curiosity what causes these to change?

Reply:
My IDE, which bloop is used for local dev, so I just left it. I'll look into it deeper, and probably pin my IDE version to make sure it's not a common thing.